src/components/FormSections/LocationDetails.tsx

```diff
@@ -76,13 +95,7 @@ const LocationDetails: React.FC = () => {
 				</Box>
 				<Box gridColumn={{ base: "1 / -1", md: "span 1" }}>
 					<FormInput labelText="Active" labelId="activeIndicatorLabel">
-						<Switch
-							id="activeIndicator"
-							name="activeIndicator"
-							disabled={isDisabled}
-							value="Active"
-							ref={register()}
-						/>
+						<Switch id="activeIndicator" name="activeIndicator" disabled={isDisabled} value="Active" ref={register} />
```

Comment:
`register` is a function that must be called to properly access a component's ref and register it with react-hook-form. Don't forget the parentheses to actually execute the function:
```suggestion
						<Switch id="activeIndicator" name="activeIndicator" disabled={isDisabled} value="Active" ref={register()} />
```